package/gluon-config-mode-geo-location/i18n/fr.po

```diff
@@ -0,0 +1,35 @@
+msgid ""
+msgstr ""
+"Content-Type: text/plain; charset=UTF-8\n"
+"Project-Id-Version: PACKAGE VERSION\n"
+"PO-Revision-Date: 2015-08-12 23:30+0100\n"
+"Last-Translator:Tobias Bernot <tqbs@airmail.cc>\n"
+"Language-Team: French\n"
+"Language: fr\n"
+"MIME-Version: 1.0\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+msgid ""
+"If you want the location of your node to be displayed on the map, you can "
+"enter its coordinates here. Specifying the altitude is optional and should "
+"only be done if a proper value is known."
+msgstr ""
+"Pour Afficher votre noeud sur la Carte nous avons besoin de ses coordonnées. "
+"Ici vous pouvez entrer sa position. La hauteur est optionelle "
```

Comment:
"hauteur" usually means the height of something, in this case the node has 2cm height :-) If it means the true GPS height from sea level, in french it's also "altitude". If it's height from street to device, I'm not sure.